tests/js/spec/views/performance/vitalDetail/index.spec.tsx

```diff
@@ -0,0 +1,252 @@
+          count: 'integer',
+          count_unique_user: 'integer',
+          team_key_transaction: 'boolean',
+          p50_measurements_cls: 'number',
+          p75_measurements_cls: 'number',
+          p95_measurements_cls: 'number',
+          project: 'string',
+          transaction: 'string',
+        },
+        data: [
+          {
+            compare_numeric_aggregate_p75_measurements_cls_greater_0_1: 1,
+            compare_numeric_aggregate_p75_measurements_cls_greater_0_25: 0,
+            count: 10000,
+            count_unique_user: 2740,
+            team_key_transaction: 1,
+            p50_measurements_cls: 0.143,
+            p75_measurements_cls: 0.215,
+            p95_measurements_cls: 0.302,
+            project: 'javascript',
+            transaction: 'something',
+          },
+        ],
+      },
+      match: [
+        (_url, options) => {
+          return options.query?.field?.find(f => f === 'p50(measurements.cls)');
+        },
+      ],
+    });
+    MockApiClient.addMockResponse({
+      method: 'GET',
+      url: `/organizations/org-slug/key-transactions-list/`,
+      body: [],
+    });
+  });
+
+  afterEach(function () {
+    MockApiClient.clearMockResponses();
+    ProjectsStore.reset();
+  });
+
+  it('renders basic UI elements', async function () {
+    mountWithTheme(<TestComponent />, {
+      context: routerContext,
+    });
+
+    // It shows a search bar
+    expect(await screen.findByLabelText('Search events')).toBeInTheDocument();
+
+    // It shows the vital card
+    expect(
+      screen.getByText(textWithMarkupMatcher('The p75 for all transactions is 4500ms'))
+    ).toBeInTheDocument();
+
+    expect(screen.getByText('Good 50%')).toBeInTheDocument();
+    expect(screen.getByText('Meh 33%')).toBeInTheDocument();
+    expect(screen.getByText('Poor 17%')).toBeInTheDocument();
+
+    // It shows a chart
+    expect(screen.getByText('Duration p75')).toBeInTheDocument();
+
+    // It shows a table
+    expect(screen.getByText('something').closest('td')).toBeInTheDocument();
+  });
+
+  it('triggers a navigation on search', async function () {
+    mountWithTheme(<TestComponent />, {
+      context: routerContext,
+    });
+
+    // Fill out the search box, and submit it.
+    userEvent.type(
+      await screen.findByLabelText('Search events'),
+      'user.email:uhoh*{enter}'
+    );
+
+    // Check the navigation.
+    expect(browserHistory.push).toHaveBeenCalledTimes(1);
+    expect(browserHistory.push).toHaveBeenCalledWith({
+      pathname: undefined,
+      query: {
+        project: 1,
+        statsPeriod: '14d',
+        query: 'user.email:uhoh*',
+      },
+    });
+  });
+
+  it('Applies conditions when linking to transaction summary', async function () {
+    mountWithTheme(<TestComponent />, {
+      context: routerContext,
+    });
+
+    expect((await screen.findByText('something')).closest('a')).toBeInTheDocument();
+  });
```

Comment:
This test is no longer asserting the same things. The original intent of this test was to assert that the anchor's href added the tag in the query param, and this no longer checks that.

Reply:
yes...we were checking the prop and now would make more sense to check the rendered `href` in the anchor element. I've to check how to do that because it does not get rendered ... even before

Reply:
found another solution :)